build.gradle

```diff
@@ -96,9 +119,15 @@ task setupServerModules(type: Sync) << {
     into(new File(rootProject.file(localServerDataPath), "modules"))
 }
 
-run.dependsOn rootProject.extractNatives
-run.dependsOn rootProject.moduleClasses
+run.workingDir = rootDir
+run.args = ["-homedir=terasology-server"]
 run.dependsOn setupServerConfig
 run.dependsOn setupServerModules
-run.workingDir = rootDir
-run.args = ["-homedir=terasology-server"]
\ No newline at end of file
+
+// Support both standalone workspace (expect certain things provided) and embedding in a Terasology workspace (use task dependencies)
+if (project.name != project(':').name) {
+    run.dependsOn rootProject.extractNatives
+    run.dependsOn rootProject.moduleClasses
+} else {
+    // For a standalone workspace natives should be provided
```

Comment:
So if I understand correctly this is the only part which is missing to have tests work in a standalone workspace, correct?
Would it work if we just copy the relevant parts (i.e. `dependencies {natives group: 'org.lwjgl.lwjgl', ...}`, `task extractNatives {...}`) from the engine's `build.gradle`, then add `dependsOn extractNatives` here?

As a (maybe better but probably more time consuming) alternative - could the code which takes care of natives be moved from the engine's `build.gradle` to somewhere accessible from the various repositories, like the `config` dir which Jenkins copies into the workspace before starting the build? Maybe it could be useful for testing modules as well? (I always worked on the engine and on this facade, but never on a module, so I don't know if they have unit tests which run during the Jenkins build)